contracts/deploy/00_generate_configuration.ts

```diff
@@ -0,0 +1,40 @@
+import { generateGenesisFile } from "../scripts/create_genesis.ts";
+import { exec } from "child_process";
+import util from "node:util";
+import path from "path";
+import fs from "fs";
+
+const CLIENT_SBIN_DIR = `${__dirname}/../../clients/geth/specular/sbin`;
+const CLIENT_DATA_DIR = `${__dirname}/../../clients/geth/specular/data`;
+const execPromise = util.promisify(exec);
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  // 1. generate genesis file
+  const dataPath = `${__dirname}/../../clients/geth/specular/data/`;
+  const baseGenesisPath = path.join(dataPath, "base_genesis.json");
+  const genesisPath = path.join(dataPath, "genesis.json");
+
+  // 2. init geth nodes
+  await execPromise(path.join(CLIENT_SBIN_DIR, "clean.sh"));
```

Comment:
I'm not sure if we actually want to run this every time we start a node. But I found it helpful for testing purposes so far

Reply:
separate this out, move this script to somewhere outside `contracts` folder. putting the script here couples L1 and L2.